api/AbortPaymentEvent.json

```diff
@@ -48,6 +48,9 @@
           "safari_ios": {
             "version_added": null
           },
+          "samsunginternet_android": {
+            "version_added": "8.0"
```

Comment:
This is a really good question we have been experimenting with Web Payment since 5.0 so I am not sure. I think to be safe I am going to say it was not and update it.